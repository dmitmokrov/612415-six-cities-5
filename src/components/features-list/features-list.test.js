import React from 'react';
import renderer from 'react-test-renderer';
import FeaturesList from './features-list';

const features = [`water`, `fire`, `bathroom`];

it(`Should FeaturesList render correctly`, () => {
  const tree = renderer
    .create(<FeaturesList
      features={features}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
