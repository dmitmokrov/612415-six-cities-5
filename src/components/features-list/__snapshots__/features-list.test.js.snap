// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FeaturesList render correctly 1`] = `
Array [
  <li
    className="property__inside-item"
  >
    water
  </li>,
  <li
    className="property__inside-item"
  >
    fire
  </li>,
  <li
    className="property__inside-item"
  >
    bathroom
  </li>,
]
`;
